src/components/donation/Steps/Submit/StripeCheckout/Checkout.tsx

```diff
@@ -61,16 +61,17 @@ export default function Checkout({ onBack }: { onBack: () => void }) {
           handleError(error, GENERIC_ERROR_MESSAGE);
         }}
       />
+
       {isLoading ? (
         <button
-          className="btn-outline-filled btn-donate w-1/2"
+          className="btn-outline-filled btn-donate w-1/2 mt-auto"
```

Comment:
min-height + auto to prevent layout shift
<img width="403" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/2ed95516-1a71-4e99-8420-d083b0e1aceb">